.github/workflows/build-test.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: "Build & Test"
-on:  # yamllint disable-line rule:truthy
+on: # yamllint disable-line rule:truthy
```

Comment:
This is double spaces on purpose. I'm pretty sure it's a yamllint rule

Reply:
Fixed, but I think we should just ignore this going forward unless there is a good reason